src/Util/Logfile/AbstractReader.php

```diff
@@ -34,11 +36,34 @@ private static function getReaders()
             return static::$readers;
         }
 
+        static::$readers = static::getCustomReaders();
```

Comment:
Why so tricky? Why not just add the reader here with `static::$readers[] = new NginxCustomLogFormatReader();`

Reply:
This allows people adding more readers without modifying the repository code, like I stated here #53